cv32e20/tb/uvmt/uvmt_cv32e20.flist

```diff
@@ -28,7 +28,8 @@
 -f ${DV_UVMA_INTERRUPT_PATH}/uvma_interrupt_pkg.flist
 -f ${DV_UVMA_OBI_MEMORY_PATH}/src/uvma_obi_memory_pkg.flist
 -f ${DV_UVMA_DEBUG_PATH}/uvma_debug_pkg.flist
--f $(DV_SVLIB_PATH)/svlib_pkg.flist
+//-f $(DV_SVLIB_PATH)/svlib_pkg.flist
+-f ../../../../../vendor_lib/verilab/svlib_pkg.flist
```

Comment:
This particular change breaks ci_check for me using xcelium.   Not sure why you went with a relative path.  I tried running ci_check from cv32e20/sim/uvmt and also from core-v-verif/ and the script couldn't find the file svlib_pkg.flist .